paths/private_groups.yaml

```diff
@@ -24,21 +27,16 @@ get:
         enum:
           - created_at
           - id
-    - name: active
-      in: query
-      description: return either active of inactive private groups
-      required: false
-      schema:
-        type: boolean
+    - $ref: '../parameters/OrderDir.yaml'
     - $ref: '../parameters/Schema.yaml'
-    - $ref: '../parameters/X-User-Token-Optional.yaml'
     - $ref: '../parameters/Bearer.yaml'
-
+    - $ref: '../parameters/X-User-Token-Optional.yaml'
```

Comment:
same here, when the x-user-token is not required, it's usually optional, unless it's completely ignored it changes the output, because of the attached public scope

Reply:
This is the same one above, just reordered the params